.github/dependabot.yml

```diff
@@ -29,6 +31,10 @@ updates:
       - dependency-name: github.com/vulcand/predicate
       # TODO(tross): Update after Firestore is updated
       - dependency-name: google.golang.org/genproto
+      # TODO(tross): Update after Firestore is updated
+      - dependency-name: cloud.google.com/go/iam
+      # TODO(tross): Update after /x/crypto is updated
+      - dependency-name: github.com/go-webauthn/webauthn
```

Comment:
Could we let these run, but rely on previous ignores/manual changes to rollback crypto?

I suppose #19579 is making this a moot point anyway.